pyaedt/edb_core/components.py

```diff
@@ -47,16 +44,17 @@ def resistor_value_parser(RValue):
 
 
 class Components(object):
-    """Components class."""
-    @property
-    def edb(self):
-        """ """
-        return self.parent.edb
+    """HFSS 3D Layout class.
```

Comment:
Again, I thought I was to use the name in class line. Why did you use HFSS 3D Layout class instead? How would  I know what to use?

Reply:
This will need more work and perhaps some help from Massimo.  Look at how `pandas` documents their `DataFrame` class:
https://pandas.pydata.org/docs/reference/frame.html
```
DataFrame([data, index, columns, dtype, copy])

Two-dimensional, size-mutable, potentially heterogeneous tabular data.
```
That's a great description.  I imagine it wouldn't be helpful to say it's the `DataFrame` class.  We need to know in one line what sort of features these classes expose.  I honestly don't know enough about AEDT to be able to provide this.